src/index.js

```diff
@@ -282,7 +282,7 @@ const main = async () => {
         const memberGroup = req.body
         // iterate through membergroups and user ids
         // Mark the task as completed for all the users
-        for (let userIds of Object.values(memberGroup)) {
+        const promises = Object.values(memberGroup).map((userIds) => {
```

Comment:
Jotta ei tulis kahta `const promises` niin olisko helpompi tehdä näin:
```
const promises = Object.values(memberGroup).map((userIds) => userIds.map((user_guid) =>
            Promise.resolve(
              postTaskEntry({
                user_guid,
                created_by: req.user.membernumber,
                task_guid: req.params.task_id,
                completion_status: 'COMPLETED',
              })
            )
          )
        })
```

Ehkä poistaa myös tarpeen tuolle `const iterablePromises = [].concat.apply([], promises)`?

Reply:
Joo!